compiler/passes/docs.cpp

```diff
@@ -84,8 +84,8 @@ void docs(void) {
     }
 
     // TODO: Check for errors here... (thomasvandoren, 2015-02-25)
-    mkdir(docsRstDir.c_str(), S_IWUSR|S_IRUSR|S_IXUSR);
-    mkdir(docsOutputDir.c_str(), S_IWUSR|S_IRUSR|S_IXUSR);
+    mkdir(docsRstDir.c_str(), 0777);
+    mkdir(docsOutputDir.c_str(), 0777);
```

Comment:
If you wanted to still use the named ones, you could combine S_IRWXU, S_IRWXG and S_IRWXO instead.  If you want to leave it like this, please leave a comment indicating the permissions used in plainspeak.